tests/test_import.py

```diff
@@ -1,11 +1,25 @@
 from pytest import mark
 
 
-def test_import_fastapi_sqla():
-    import fastapi_sqla  # noqa
+def test_import_sync_api():
+    from fastapi_sqla import (  # noqa
+        Base,
+        Collection,
+        Item,
+        Page,
+        Paginate,
```

Comment:
what about `PageSignature`?

Reply:
`PaginateSignature` is actually not used externally